dts: arc: quark_se_c1000_ss: Fix worng interrupt number in i2c 0/1

SavinayDharmappa opened the pull request:
This fixes https://github.com/zephyrproject-rtos/zephyr/issues/7026

The issue is due worng interrupt number populated in dts for
i2c0 and i2c1 instance.

Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>

ydamigos reviewed (approved):
LGTM. I checked them against quark_se SoC in ext/hal.

SavinayDharmappa committed:
dts: arc: quark_se_c1000_ss: Fix worng interrupt number in i2c 0/1

This fixes https://github.com/zephyrproject-rtos/zephyr/issues/7026

The issue is due worng interrupt number populated in dts for
i2c0 and i2c1 instance.

Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>
